src/dotnet/commands/dotnet-build/Program.cs

```diff
@@ -93,10 +104,53 @@ public static int Run(string[] args)
 
                 msbuildArgs.AddRange(app.RemainingArguments);
 
-                return new MSBuildForwardingApp(msbuildArgs).Execute();
+                return 0;
             });
 
-            return app.Execute(args);
+            int exitCode = app.Execute(args);
+            if (msbuildArgs == null)
```

Comment:
All right, if @jonsequitur and you are tracking fixing this soon. Seems all right by me.